src/bullets/data_storage/cache_storage.py

```diff
@@ -0,0 +1,55 @@
+from enum import Enum
+from appdirs import *
+
+from bullets.data_storage.endpoints.cache_price import CachePrice
+from bullets.data_storage.endpoints.cache_statement import CacheStatement
+
+
+class CacheEndpoint(Enum):
```

Comment:
I'm not sure `CacheEndpoint` is the best name for this enum, it might make people think that we have endpoint with prices cached. Maybe `LocalCache` would be better?

Reply:
Totally right, changed